### PR TITLE
Task4

### PR DESCRIPTION
**Pull request by yana-may25**

### CODE REVIEW

File: src/task4/task4.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+function countChar(str, char){
```
Comment (MrOlegKravchenko) [inline src/task4/task4.js:1]: There is also should be **countBs** function.
Function writing pattern please.
AND lets make one importent rule:
"After each new **{** should be a new 1 tab right line (or 4 scapes, whatever)"

File: src/task4/task4.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    if (typeof(str)=== "string" && typeof(char)=== "string" && char.length===1){
```
Comment (MrOlegKravchenko) [inline src/task4/task4.js:2]: **countChar** doesn't need this **if**.

File: src/task4/task4.js (hunk 1 above)
Comment (yana-may25) [reply src/task4/task4.js]: my bad

File: src/task4/task4.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    let count = 0;
+    for (let i = 0; i < str.length; i++)
+        if (str.charAt(i) === char)
+            count++
+    return count;
+};
+
+function countBs(str) {
+    return countChar(str, "B")
+}
```
Comment (MrOlegKravchenko) [inline src/task4/task4.js:11]: **Brilliant!**

The way I like is:
```
function countChar(str, char) {
    let count = 0;
    for (let i = 0; i < str.length; i++) {
        if (str.charAt(i) === char) {
            count++;
        }
    }
    return count;
};

function countBs(str) {
    return countChar(str,'B');
};
```

**Review by MrOlegKravchenko — APPROVED** [review]

**Pull request merged**